papers/ring_vrf/zk_cont.tex

```diff
@@ -1,34 +1,101 @@
-
-\section{Zero-knowledge continuations}
+\section{Zero-knowledge Continuations}
 \label{sec:rvrf_cont}
 
-We now develop our zero-knowledge continuation technique, called 
-{\em special(ized) Groth16} or \SpecialG, and
- build ring VRFs with fast amortized prover time.
-Anonymity demands we finalize the amortized ``continuation'' multiple
-times, with each time being unlinkable to the others, meaning our rVRF
-stays zero-knowledge even with the continuation being reused.
-
-Our zero-knowledge continuation $\SpecialG_\rel$ consists of three algorithms:
+\noindent In the following, we describe a NIZK for a relation $\rel$ where
```

Comment:
Maybe we can add something to explain why we \reltwo here. For example, we can start the section by saying something like this:
'Given that we have a NIZK for the relation \rel, we divide the relation \rel in two relations \relone and \reltwo. \relone corresponds to the expensive part the  proving algorithm of NIZK for \rel.'